spec/features/sign_in_spec.rb

```diff
@@ -43,7 +43,7 @@
 
     # Redirect to DfE Signin and come back
     expect(page).to have_content("Sign out (#{user.first_name} #{user.last_name})")
-    expect(root_page).to be_displayed
+    expect(page.current_path).to eql("/rollover")
```

Comment:
At first I was confused how this was passing then saw that the user's next state will redirect the user to roll_over path. 👍 
`accepted_rollover_2021: rollover_path`